### ISSUE TITLE
Iris flower classification

### ISSUE THREAD

**Issue by A-GHOSH-dev** [IEEE-WIE-VIT/WIE-WoC#175]
https://github.com/IEEE-WIE-VIT/WIE-WoC/tree/main/ML%20AI

**Comment by UrviSinghal28** [comment]
Please assign this to me

**Comment by UrviSinghal28** [comment]
hey you can close it, i guess it merged with my previous pull request so already updated

**Comment by A-GHOSH-dev** [comment]
Have you completed this PR?

**Comment by urvisinghal** [comment]
yes its already merged with the main repo, got merged along with the previous issue
